README.rst

```diff
@@ -39,8 +80,10 @@ Translation
     pybabel extract alarmbot/*.py -o locales/alarmbot.pot
     pybabel compile -d locales -l vi -D alarmbot
 
+
 API
 ===
+We need  API tools in order to test API in projects.In this project , i use `Postman <https://www.postman.com/downloads/>`_ to test API.
```

Comment:
No need to specify this. Anyone working with web need to have knowledge about HTTP and should know some HTTP tools.